index.d.ts

```diff
@@ -0,0 +1,2148 @@
+  address_validation_level: string;
+
+  /**
+   * Set this option to true if your shipment contains alcohol.
+   *  - UPS - only supported for US Domestic shipments
+   *  - FedEx - only supported for US Domestic shipments
+   *  - Canada Post - Requires adult signature 19 years or older. If you want adult signature 18 years or older, instead use delivery_confirmation: ADULT_SIGNATURE
+   */
+  alcohol: boolean;
+
+  /**
+   * Setting this option to true will indicate to the carrier to prefer delivery by drone, if the carrier supports drone delivery.
+   */
+  by_drone: boolean;
+
+  /**
+   * Setting this to true will add a charge to reduce carbon emissions.
+   */
+  carbon_neutral: boolean;
+
+  /**
+   * Adding an amount will have the carrier collect the specified amount from the recipient.
+   */
+  cod_amount: string;
+
+  /**
+   * Method for payment. "CASH", "CHECK", "MONEY_ORDER"
+   */
+  cod_method: "CASH" | "CHECK" | "MONEY_ORDER";
+
+  /**
+   * The ID of the Address to which the COD payment should be returned. 
+   * Defaults to the origin address. 
+   * Only available on FedEx shipments.
+   */
+  cod_address_id: string;
+
+  /**
+   * Which currency this shipment will show for rates if carrier allows.
+   */
+  currency: string;
+
+  /**
+   * If you want to request a signature, you can pass "ADULT_SIGNATURE" or "SIGNATURE". 
+   * You may also request "NO_SIGNATURE" to leave the package at the door.
+   *  - All - some options may be limited for international shipments
+   *  - FedEx - "INDIRECT_SIGNATURE" is also an option
+   *  - USPS - additional options
+   *    - "ADULT_SIGNATURE_RESTRICTED"
+   *    - "SIGNATURE_RESTRICTED"
+   */
+  delivery_confirmation: "ADULT_SIGNATURE" | "SIGNATURE" | "NO_SIGNATURE" | "INDIRECT_SIGNATURE" | "ADULT_SIGNATURE_RESTRICTED" | "SIGNATURE_RESTRICTED";
+
+  /**
+   * Method the customer will use to transfer the package to the carrier. 
+   * "REGULAR_PICKUP", "SCHEDULED_PICKUP", "RETAIL_LOCATION", "STATION" or "DROP_BOX". 
+   * Supported carriers and their corresponding carrier dropoff codes:
+   *  - FedEx
+   *    - "REGULAR_PICKUP" - "REGULAR_PICKUP" (default)
+   *    - "SCHEDULED_PICKUP" - "REQUEST_COURIER"
+   *    - "RETAIL_LOCATION" - "BUSINESS_SERVICE_CENTER"
+   *    - "STATION" - "STATION"
+   *    - "DROP_BOX" - "DROP_BOX"
+   */
+  dropoff_type: "REGULAR_PICKUP" | "SCHEDULED_PICKUP" | "RETAIL_LOCATION" | "STATION" | "DROP_BOX" | "REQUEST_COURIER" | "BUSINESS_SERVICE_CENTER";
+
+  /**
+   * Package contents contain dry ice.
+   *  - UPS - Need dry_ice_weight to be set
+   *  - UPS MailInnovations - Need dry_ice_weight to be set
+   *  - FedEx - Need dry_ice_weight to be set
+   */
+  dry_ice: boolean;
+
+  /**
+   * If the dry ice is for medical use, set this option to true.
+   *  - UPS - Need dry_ice_weight to be set
+   *  - UPS MailInnovations - Need dry_ice_weight to be set
+   */
+  dry_ice_medical: boolean; // TODO docs say string but description indicates boolean value
```

Comment:
The [docs](https://www.easypost.com/docs/api/node#options-object) have **dry_ice_medical** type as **string** but description indicates it should be **boolean** value.

> If the dry ice is for medical use, set this option to **true**.

Reply:
@Justintime50 Is the [Shipment Options Object](https://www.easypost.com/docs/api/node#options-object) property `dry_ice_medical` a string or boolean?

![image](https://user-images.githubusercontent.com/2395597/157888766-395b05f1-665b-4102-8780-66cb3e6361dc.png)

Reply:
Heh, it's a bool for sure. I'll get this corrected internally